<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userWindow</class>
 <widget class="QDialog" name="userWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>189</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="id">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>139</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>99</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="isId">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="isUsername">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>79</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter an Admin Account Information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>210</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Add User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>210</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>210</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>229</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="usersDisplay">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>561</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="displayAll">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>109</y>
     <width>220</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Display All Users</string>
   </property>
  </widget>
  <widget class="QPushButton" name="displayBorrowin">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>220</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Display Borrowing Users</string>
   </property>
  </widget>
  <widget class="QPushButton" name="displayReserving">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>220</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Display Reserving Users</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>210</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Script MT</family>
     <pointsize>30</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>User Management</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>79</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Users: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>169</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add a New User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit an Existing User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Remove an Existing User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Go Back to The Main Window</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>490</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Display User Reserving a Resource</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Display Users Borrowing a Resource</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Display All Isers (Refresh the Display)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
